ProjectK.Business/Characters/Character.cs

```diff
@@ -37,6 +38,10 @@ public class Character : ICharacterEdit
 
         // public ICollection<Skill> Skills { get; } = new HashSet<Skill>();
 
+        public int Level { get; private set; }
+        public long TotalXp { get; set; }
```

Comment:
Why is the setter not private here?

Reply:
I think there are more setters like this one